### PR TITLE
Ux_issues

### PR DESCRIPTION
**Pull request by elhadis**
## What does this PR do?

I asked for UX issues #8562  to fix , but I did not get any permission,

### CODE REVIEW

**Comment by Udit-takkar** [comment]
hey @elhadis please can you elaborate more about this PR? I am not able to understand what have you issue have you fixed here

File: .env.example
Comment (sean-brydon) [inline .env.example:1]: Can you revert the changes here please :) We dont want these changes in our repo

**Comment by sean-brydon** [comment]
Hi @elhadis thanks for the contribution - I'm not exactly sure what this PR is trying to do but it doesnt seem to be link to the issue you mentioned in the body. 

I'm going to close this for now - if you believe i am mistaken please reopen.

The issue you mentioned is our website repo which is not public. Thanks
